instrumentation/runtime-telemetry/runtime-telemetry-java8/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java8/RuntimeMetrics.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.runtimemetrics.java8;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.runtimemetrics.java8.internal.JmxRuntimeMetricsUtil;
+import java.io.Closeable;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/** The entry point class for runtime metrics support using JMX. */
+public final class RuntimeMetrics implements Closeable {
```

Comment:
(minor)

Having two `RuntimeMetrics` classes (Java 8 and Java 17) has confused me.

Perhaps renamed this one into `Java8RuntimeMetrics`? 

Same comment for `RuntimeMetricsBuilder`.

Reply:
these classes were not introduced by this PR - just FYI

Reply:
This PR adds `RuntimeMetrics` and `RuntimeMetricsBuilder` classes for Java 17. So, these class names exist both for the Java 8 and Java 17 runtime metrics implementations.

Reply:
ok - renamed